src/vs/workbench/contrib/debug/browser/breakpointWidget.ts

```diff
@@ -114,6 +116,8 @@ export class BreakpointWidget extends ZoneWidget implements IPrivateBreakpointWi
 				this.context = Context.LOG_MESSAGE;
 			} else if (this.breakpoint && !this.breakpoint.condition && this.breakpoint.hitCondition) {
 				this.context = Context.HIT_COUNT;
+			} else if (this.breakpoint && this.breakpoint.waitFor) {
+				this.context = Context.WAIT_FOR_BREAKPOINT;
```

Comment:
I'm okay with embracing the name "trigger points". In that case this could be `breapoint.triggeredBy` / `Context.TRIGGER_POINT`